cirq/circuits/circuit.py

```diff
@@ -1741,12 +1749,20 @@ def _get_operation_circuit_diagram_info_with_fallback(
     # Fallback to a default representation using the operation's __str__.
     name = str(op)
 
+    # For TaggedOperation, use the sub_operations __str__ instead
+    if isinstance(op, cirq.TaggedOperation):
+        name = str(op.sub_operation)
+
     # Representation usually looks like 'gate(qubit1, qubit2, etc)'.
     # Try to cut off the qubit part, since that would be redundant information.
     redundant_tail = '({})'.format(', '.join(str(e) for e in op.qubits))
     if name.endswith(redundant_tail):
         name = name[:-len(redundant_tail)]
 
+    # Add tags onto the representation, if they exist
+    if isinstance(op, cirq.TaggedOperation):
+        name += f' {list(op.tags)}'
```

Comment:
Is the extra space intentional?  The other version just appended it without a space.

Reply:
Changed.